CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
-## [0.60.0] - 2020-04-07
+## [0.60.1] - 2021-04-09
+### Updated
```

Comment:
If we follow `keepachangelog` format this can either be Added/Removed/Changed/Fixed so in this case I suppose can all go under Changed